### PR TITLE
com.jme3.anim.tween.action.BaseAction: basic javadoc

### PR DESCRIPTION
**Pull request by Scrappers-glitch**
This PR provides basic javadoc for the class `com.jme3.anim.tween.action.BaseAction`.

### CODE REVIEW

File: jme3-core/src/main/java/com/jme3/anim/tween/action/BaseAction.java [hunk 1]
```diff
@@ -35,14 +35,38 @@
 import com.jme3.anim.tween.ContainsTweens;
 import com.jme3.anim.tween.Tween;
 import com.jme3.util.SafeArrayList;
-
 import java.util.List;
 
+/**
+ * A simple implementation for the interface {@link Action} to provide a wrapper for a {@link Tween}.
```
Comment (Ali-RS) [inline jme3-core/src/main/java/com/jme3/anim/tween/action/BaseAction.java:41]: Note that `Action` is an "abstract class" to be specific.

File: jme3-core/src/main/java/com/jme3/anim/tween/action/BaseAction.java [hunk 2]
```diff
@@ -97,8 +97,10 @@ public void setMaskPropagationEnabled(boolean maskPropagationEnabled) {
     }
 
     /**
-     * Applies an animation mask to this action, if the {@link BaseAction#maskPropagationEnabled} is 
-     * enabled, the animation mask is propagated to the child actions (or wrapped actions).
+     * Internal use only.
+     *
+     * Dispatched from {@link com.jme3.anim.AnimLayer#update(float)}. Applies an animation mask to this action, 
+     * if the {@link BaseAction#maskPropagationEnabled} is enabled, the animation mask is propagated to the child actions (or wrapped actions).
      *
```
Comment (Scrappers-glitch) [inline jme3-core/src/main/java/com/jme3/anim/tween/action/BaseAction.java:104]: I marked this as an internal use-only method, and clarified it's being called from the `AnimLayer#update(...)`.

**Comment by Scrappers-glitch** [comment]
I added some refinements and simplifications. It's ready for another review.

**Comment by Scrappers-glitch** [comment]
Thanks for merging this PR. I am glad my work is appreciated. I have been working on the animation system documentation for a long time, and now preparing to maintain the API by further studying the architecture involved.

**Pull request merged**